build.sbt

```diff
@@ -18,7 +18,7 @@ lazy val core =
   project
     .enablePlugins(AutomateHeaderPlugin)
     .configs(MultiJvm)
-    .dependsOn(coordination,`coordination-etcd` % "test->compile")
+    .dependsOn(coordination,`coordination-etcd` % "test->test")
```

Comment:
That's because we use `DockerEtcdService` also in `core`. And since `DockerEtcdService` is defined in `coordination-etcd` test scope we need this change